cilium/cmd/bpf_ct_list.go

```diff
@@ -29,7 +29,7 @@ import (
 
 // bpfCtListCmd represents the bpf_ct_list command
 var bpfCtListCmd = &cobra.Command{
-	Use:     "list ( <endpoint identifier> | global )",
+	Use: "list ( <endpoint identifier> | global )",
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
File is not `gofmt`-ed with `-s` (gofmt)